app/views/admin/creator_settings/_form.html.erb

```diff
@@ -78,3 +78,21 @@
     </div>
   </fieldset>
 </div>
+
+<div class="crayons-field mt-6 align-left">
+  <fieldset aria-describedby="section-description">
+    <legend class="crayons-field__label mb-2">Finally, please agree to the following:</legend>
```

Comment:
👋  @ludwiczakpawel 

These checkboxes weren't part of the original creator onboarding design, so I did my best to make them pretty. That said, I reused copy from the user onboarding COC and TOS checkboxes for both of these COC and TOS checkboxes. What I was unsure of though was what copy to use, if any, here as the checkbox group's legend. Any suggestions?

Reply:
Update: I spoke with Jennie and she said that the copy itself looked good, but suggested that I add a note about the COC and TOS for Creators beneath it. That being said, I've added the following:
![Screen Shot 2021-11-19 at 9 39 24 AM](https://user-images.githubusercontent.com/32834804/142659215-3f95dfa8-ee25-4ae3-9369-af07eb035c5d.png)
cc @ludwiczakpawel